src/main.cpp

```diff
@@ -159,7 +159,7 @@ int StressTest() {
     std::ifstream biome_file = files::open_data_file("biome_data.json");
     biome_file >> biome_data;
 
-    World world(materials_json, biome_data, 12, 12);
+    World world(materials_json, biome_data, 2, 2);
```

Comment:
Make these into `constexpr`s

Reply:
No the 2s

Reply:
it's literally an r value. I don't think this will change anything.

Reply:
Good practice for explaining what arbitrary constants mean.